src/content/app-platform/getting-started/index.md

```diff
@@ -0,0 +1,260 @@
+
+NAME                       VERSION   LAST DEPLOYED   STATUS
+cert-exporter              1.6.1     69m             deployed
+cert-manager               2.4.3     68m             deployed
+chart-operator             2.13.0    68m             deployed
+cluster-autoscaler         1.19.2    69m             deployed
+coredns                    1.4.1     67m             deployed
+external-dns               2.3.0     69m             deployed
+grafana                    0.2.0     46m             deployed
+kiam                       1.7.1     67m             deployed
+kube-state-metrics         1.3.1     69m             deployed
+metrics-server             1.2.2     69m             deployed
+net-exporter               1.9.3     69m             deployed
+node-exporter              1.7.2     68m             deployed
+```
+
+## Finding the ingress controller version
+
+You can browse the apps in our catalog using our [web UI]({{< relref "/ui-api/web" >}})
+but this information is also available in the management cluster. We create
+[AppCatalogEntry]({{< relref "/ui-api/management-api/crd/appcatalogentries.application.giantswarm.io.md" >}})
+CRs for the apps that are available.
+
+In this example we are filtering using the `latest=true` label to show only the
+latest version according to semantic versioning.
+
+We are also filtering using the catalog label. You can change this to
+`giantswarm-playground` to show our more experimental apps which we encourage
+you to try but do not offer as a managed service.
+
+```sh
+kubectl get appcatalogentry -l latest=true,application.giantswarm.io/catalog=giantswarm
+
+NAME                                             CATALOG      APP NAME                       APP VERSION   VERSION        AGE
+...
+giantswarm-nginx-ingress-controller-app-1.16.1   giantswarm   nginx-ingress-controller-app   v0.45.0       1.16.1         25h
+...
+```
+
+## Creating an App CR
+
+We can use the [kubectl gs template app]({{< relref "/ui-api/kubectl-gs/login" >}})
+command to generate the App CR using the latest version from the previous command.
+
+```sh
+kubectl gs template app \
+  --catalog=giantswarm \
+  --cluster=${NAMESPACE} \
+  --name=nginx-ingress-controller-app \
+  --namespace=kube-system \
+  --version=1.16.1 > nginx-ingress-controller-app.yaml
+
+kubectl apply -f nginx-ingress-controller-app.yaml
+```
+
+Lets first see the output of the template command which shows only the
+required fields.
+
+```yaml
+apiVersion: application.giantswarm.io/v1alpha1
+kind: App
+metadata:
+  name: nginx-ingress-controller-app
+  namespace: s2wf9
+spec:
+  catalog: giantswarm
+  kubeConfig:
+    inCluster: false
+  name: nginx-ingress-controller-app
+  namespace: kube-system
+  version: 1.16.1
+```
+
+## Defaulting and App Status
+
+Now lets check the app using the `kubectl gs get app` command which filters
+the output to show only the most important fields.
+
+```sh
+kubectl -n ${NAMESPACE} gs get app nginx-ingress-controller-app -o yaml
```

Comment:
I think this will be useful for customers but mainly I want it for myself 😅 

I use the `kubectl neat` plugin to get rid of the managed fields and other stuff I don't care about but it also removes the CR status which is a pain.

https://github.com/itaysk/kubectl-neat